cborg-json/cborg-json.cabal

```diff
@@ -44,7 +44,7 @@ benchmark bench
 
   default-language:  Haskell2010
   ghc-options:
-    -Wall -rtsopts -fno-cse -fno-ignore-asserts -fno-warn-orphans
+    -Wall -rtsopts -fno-cse -fno-ignore-asserts -fno-warn-orphans -O2
```

Comment:
I think this patch name was meant to apply to the next patch.

Not that it matters. We can rebase and squash the whole lot.